Neos.Media/Configuration/Settings.yaml

```diff
@@ -72,6 +72,7 @@ Neos:
         # Image quality, from 0 to 100
         quality: 90
         convertCMYKToRGB: true
+        interlace: '%\Imagine\Image\ImageInterface::INTERLACE_NONE%'
```

Comment:
Not sure about that i like using a constand of a library here. Why not just add "none" explain the options as with quality above.

Reply:
Consistant with the Filter configuration

Reply:
You are right